File: Python/1012.py
import math

line = input()
a, b, c = line.split(' ')
a = float(a)
b = float(b)
c = float(c)

area_a = a*c/2
area_b = 3.14159 * math.pow(c, 2)
area_c = (a + b) * c/2 
area_d = math.pow(b, 2)
area_e = a*b

print('TRIANGULO: {:.3f}'.format(area_a))
print('CIRCULO: {:.3f}'.format(area_b))
print('TRAPEZIO: {:.3f}'.format(area_c))
print('QUADRADO: {:.3f}'.format(area_d))
print('RETANGULO: {:.3f}'.format(area_e))

File: JavaScript/1012.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var line = lines.shift();
line = line.split(' ')

var a = parseFloat(line[0])
var b = parseFloat(line[1])
var c = parseFloat(line[2])

var area_a = a*c/2
var area_b = 3.14159 * Math.pow(c, 2)
var area_c = (a + b) * c/2 
var area_d = Math.pow(b, 2)
var area_e = a*b

console.log('TRIANGULO: ' + area_a.toFixed(3))
console.log('CIRCULO: ' + area_b.toFixed(3))
console.log('TRAPEZIO: ' + area_c.toFixed(3))
console.log('QUADRADO: ' + area_d.toFixed(3))
console.log('RETANGULO: ' + area_e.toFixed(3))